projects/js-toolkit/packages/js-toolkit-core/src/format.ts

```diff
@@ -92,13 +93,13 @@ export function error(
  * ```
  *
  * @remarks
- * Error messages are prepended with a 'ℹ️' emoji.
+ * Error messages are prepended with a '🛈 ' emoji.
```

Comment:
```suggestion
 * Info messages are prepended with a '🛈 ' emoji.
```

and 

<img width="1304" alt="Screen Shot 2021-06-03 at 13 08 56" src="https://user-images.githubusercontent.com/7663513/120676872-db519100-c46c-11eb-8db9-837dd011f338.png">

Looks like this emoji don't exists on MacOS Big Sur 11.3.1 (20E241)

Reply:
Thanks @diegonvs, that's one of the things I wanted to check, because I have lots of fonts with emojis installed in my system.

I'm not sure if we should remove them, because not everybody has fonts with emoji support and, when they are not available, the display looks quite bad.

I added the possibility to define an environment variable to prevent emojis from being shown, but I still find it a bit complex to configure for a very trivial functionality. On the other hand, the previous emojis we had have been a long time lying around and nobody has complaint, so it may be that they are standard enough...

@bryceosterhaus what do you think? Should we remove them or revert them to the previous ones (uglier but seemingly standard) and wait to see what happens when we release the tool?

Reply:
Note that emoji signs make the messages easier to read, so they are not completely useless. But still, it's not a drama if we don't have them, I guess :shrug:

Reply:
I have pushed-forced without the commit that changed emojis. We will be using the old ones that worked correctly at least in Mac and my Linux box.

Reply:
Personally, I like it when there are helpful and good looking emojis. However, if there is a case where we might see `[]`, I'd prefer to just keep our old ones, unless there is a way to easily provide a fallback.

Reply:
It really depends on what fonts you have installed. There's a project, for example, to create a font of the fly with loads of icons :point_right: https://github.com/sebastiencs/icons-in-terminal#overview.

In general, I've found that the typical ones we use (which usually are 1 char wide and monochrome on Linux) are also available in Mac. However, it's -almost- impossible to know if a certain emoji will be displayed in a terminal, because you need to know the font in use and then check that font to see if the icon is there (unfeasible).

But I would say we are safe with the old ones...